app/scripts/filters/resources.js

```diff
@@ -1274,15 +1274,36 @@ angular.module('openshiftConsole')
       });
     };
   })
-  .filter('scopeDetails', function() {
+  .filter('scopeDetails', function(sentenceCaseFilter) {
     var scopeMessages = {
       "Terminating": "Matches pods that have an active deadline.",
       "NotTerminating": "Matches pods that do not have an active deadline.",
       "BestEffort": "Matches pods that have best effort quality of service.",
       "NotBestEffort": "Matches pods that do not have best effort quality of service."
     };
     return function(scope) {
-      return scopeMessages[scope];
+      return scopeMessages[scope] || sentenceCaseFilter(scope);
+    };
+  })
+  .filter('isBestEffortPod', function(isNilFilter, usageValueFilter) {
+    return function(pod) {
+      // To be best effort a pod must not have any containers with any non-zero requests or limits
+      // Break out as soon as we find any pod with a non-zero request or limit
+      return !_.some(pod.spec.containers, function(container){
+        var hasNonZeroRequest = _.some(_.get(container, 'resources.requests'), function(request){
+          return !isNilFilter(request) && usageValueFilter(request) !== 0;
+        });
+        var hasNonZeroLimit = _.some(_.get(container, 'resources.limits'), function(limit){
+          return !isNilFilter(limit) && usageValueFilter(limit) !== 0;
+        });
+        return hasNonZeroRequest || hasNonZeroLimit;
+      });
+    };
+  })
+  .filter('isTerminatingPod', function(isNilFilter) {
```

Comment:
We have an existing `isTerminating` filter, but with different logic.

https://github.com/spadgett/origin-web-console/blob/e8e3ecdc1dbe23297674577da7180d48b0c6a3c2/app/scripts/filters/resources.js#L850-L854

Should we update the existing filter instead?